tools/mpremote/mpremote/pyboardextended.py

```diff
@@ -26,6 +26,7 @@
 fs_hook_code = """\
 import uos, uio, ustruct, micropython, usys
 
+SEEK_SET = micropython.const(0)
```

Comment:
This won't work as expected (it won't optimise the const).  Instead just do `SEEK_SET = 0`